zmon_worker_monitor/zmon_worker/tasks/main.py

```diff
@@ -327,8 +327,11 @@ def alert_series(f, n, con, check_id, entity_id):
     Evaluate given function on the last n check results and return true if the "alert" function f returns true for
     all values
     """
-
     vs = get_results(con, check_id, entity_id, n)
+
+    if len(vs) < n:
+        return False
```

Comment:
hmm, this should also not be the case --- we still need to evaluate the function on every value